src/services/database.py

```diff
@@ -14,20 +15,194 @@ def get_db_connection(db_file: str):
     con.row_factory = sqlite3.Row
     return con
 
-def initialize_users_db(erase=False) -> None:
+def initialize_users_db(reset: bool = False) -> None:
```

Comment:
This strategy ( "reset" / "erase" ) as an "optional" parameter, can very quickly result in data being deleted. The "delete" operation should be explicit, and used only from places where it is necessary.